src/neuroconv/nwbconverter.py

```diff
@@ -112,7 +112,9 @@ def add_to_nwbfile(self, nwbfile: NWBFile, metadata, conversion_options: Optiona
         conversion_options = conversion_options or dict()
         for interface_name, data_interface in self.data_interface_objects.items():
             data_interface.add_to_nwbfile(
-                nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
+                nwbfile=nwbfile,
+                metadata=metadata,
+                conversion_options=conversion_options.get(interface_name, dict()),
```

Comment:
```suggestion
                nwbfile=nwbfile, metadata=metadata, **conversion_options.get(interface_name, dict())
```

Never mind, this isn't going to word after all - `add_to_nwbfile` takes dynamic kwargs by design https://github.com/catalystneuro/neuroconv/blob/add_MiniscopeConverterPipe/src/neuroconv/basedatainterface.py#L55

like in https://github.com/catalystneuro/neuroconv/blob/main/src/neuroconv/datainterfaces/ecephys/baserecordingextractorinterface.py#L268-L275